typings/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "typings",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "type-check": "tsc -p ./tsconfig.json"
```

Comment:
does this need to be called in CI?

Reply:
it is already called  
![2021-11-03 at 18 44](https://user-images.githubusercontent.com/1223799/140163684-2de40bdc-d741-407c-8cbb-85600100a3fb.png)